core/register_core_types.cpp

```diff
@@ -226,19 +226,19 @@ void register_core_types() {
 	GDREGISTER_CLASS(PCKPacker);
 
 	GDREGISTER_CLASS(PackedDataContainer);
-	GDREGISTER_VIRTUAL_CLASS(PackedDataContainerRef);
+	GDREGISTER_ABSTRACT_CLASS(PackedDataContainerRef);
 	GDREGISTER_CLASS(AStar);
 	GDREGISTER_CLASS(AStar2D);
 	GDREGISTER_CLASS(EncodedObjectAsID);
 	GDREGISTER_CLASS(RandomNumberGenerator);
 
-	GDREGISTER_VIRTUAL_CLASS(ResourceImporter);
+	GDREGISTER_ABSTRACT_CLASS(ResourceImporter);
```

Comment:
Isn't that something that might typically make sense to implement as virtual?

Reply:
Yes, I completely forgot about these, I will add them.

Reply:
Looking at them, this will require likely another PR because its a lot of code.